tests/py3/test_annotations.py

```diff
@@ -17,10 +17,90 @@
 
 from __future__ import division, print_function, absolute_import
 
+import sys
+import inspect
+
+import pytest
+
 import hypothesis.strategies as st
 from hypothesis import given
+from hypothesis.errors import InvalidArgument
+from hypothesis.internal.compat import FullArgSpec, getfullargspec
+from hypothesis.internal.reflection import define_function_signature, \
+    convert_positional_arguments, get_pretty_function_description
 
 
 @given(st.integers())
 def test_has_an_annotation(i: int):
     pass
+
+
+@pytest.mark.skipif(sys.version_info[:2] == (3, 5),
+                    reason='getargspec was briefly deprecated, so we wrap it')
+def test_inspection_compat():
```

Comment:
Doesn't it have to run under python 2? I thought a place under the /py3 tests made more sense, but happy to move if you have a better idea than just skipping under py2.